### PR TITLE
Feature user menu

### PR DESCRIPTION
**Pull request by lbmerchant93**
## *Purpose:*

- [x] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*

Creates the User Menu that is displayed at the top of the Profile and Orders components. Creates the Orders component with a simple message to display that the component and route that was created for it both work. 

## *What are the relevant tickets?*
 
Closes 52

## *Bugs:*

Orders component still needs all of the different elements that are to be displayed. Profile has the preview commented out code still in order to explain what what changed. 

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change
- [x] I used a rebase workflow

## *Notes:*
Need information on how the API response will look in order to display the correct Orders information, can put in some sample data if we want to.

### CODE REVIEW

**Comment by christina-cook** [comment]
This menu looks good. Buttons are routing to the correct paths. Now we can style the Orders component and add the rest of the elements needed on the profile page.

File: code/web/src/modules/user/common/Menu.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+UserMenu.propTypes = {
+    logout: PropTypes.func.isRequired
+}
+
+// Component State
+function userMenuState(state) {
+    return {
+      user: state.user
+    }
+  }
```
Comment (mainlyetcetera) [inline code/web/src/modules/user/common/Menu.js:42]: Super, super picky from me. This `}` is still 2 spaces out from the edge. Looks like these docs have a mix of 2-space indents and 4. We will want to match where we see it, I think.

File: code/web/src/modules/user/Orders.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const Orders = () => (
+    <div>
+        {/* Top menu bar */}
+        <UserMenu />
+
+        <h1>Dummy Component Pure</h1>
+    </div>
+)
+
+export default Orders;
```
Comment (mainlyetcetera) [inline code/web/src/modules/user/Orders.js:17]: I advocate for removing the `;` here. Looks like, throughout the codebase, the original authors opted not to use them.

**Review by mainlyetcetera — CHANGES_REQUESTED** [review]
No issues with the code itself. Just real picky stuff from me about spacing and punctuation.

**Review by christina-cook — APPROVED** [review]
Code looks good and menu buttons are routing to the correct paths. Now we can add the rest of the elements needed on the profile page.

File: code/web/src/modules/user/common/Menu.js (hunk 1 above)
Comment (mainlyetcetera) [reply code/web/src/modules/user/common/Menu.js:41]: This looks much better. Thanks for addressing this!

**Review by mainlyetcetera — APPROVED** [review]

**Pull request merged**